core/mybatis-generator-core/src/main/java/org/mybatis/generator/api/dom/java/Method.java

```diff
@@ -223,7 +224,11 @@ public String getFormattedContent(int indentLevel, boolean interfaceMethod) {
                     comma = true;
                 }
 
-                sb.append(fqjt.getShortName());
+                if(compilationUnit.getImportedTypes().contains(fqjt)) {
+                    sb.append(fqjt.getShortName());
+                } else {
+                    sb.append(fqjt.getFullyQualifiedName());
+                }
             }

```

Comment:
Probably forgot to do it :-) - thanks for the hint.

Zitat von Jeff Butler <notifications@github.com>:

>> @@ -223,7 +224,11 @@ public String getFormattedContent(int  
>> indentLevel, boolean interfaceMethod) {
>>                      comma = true;
>>                  }
>>
>> -                sb.append(fqjt.getShortName());
>> +                if(compilationUnit.getImportedTypes().contains(fqjt)) {
>> +                    sb.append(fqjt.getShortName());
>> +                } else {
>> +                    sb.append(fqjt.getFullyQualifiedName());
>> +                }
>>              }
>
> Why is this not a call to your `addTypeName` method also?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mybatis/generator/pull/63/files#r50047524